frontend2/src/components/team/ScrimmageSettings.tsx

```diff
@@ -101,6 +117,16 @@ const ScrimmageSettings: React.FC = () => {
               team.profile?.auto_accept_ranked ?? false,
             )}
             onChange={(checked) => {
+              // TODO: does this object get passed in automatically? Can we just invoke the mutate function without passing in the object?
+              updateTeam({
+                episodeId,
+                patchedTeamPrivateRequest: {
+                  profile: {
+                    ...team.profile,
```

Comment:
we shouldn''t need these b/c it's a partial update, similar to the one in EligibiilitySettings
```suggestion
                 
```